README.md

```diff
@@ -23,27 +23,32 @@ Binaries are available on the [releases](https://github.com/chanzuckerberg/bless
 
 ## Usage
 
+At a high level:
+1. [Install](#install) blessclient
+1. If you don't have an SSH key, generate one with `ssh-keygen -t rsa -b 4096`
+1. [Import](#import-config) or [generate](#init) a blessclient config
+1. Run `blessclient run` and make sure there are no errors
+1. Modify your [ssh config](#sshconfig) to be bless compatible
+1. ssh, scp, rsync as you normally would
+
 ### Config
 
 By default, `blessclient` looks for configs in `~/.blessclient/config.yml`. You can always override this `blessclient run -c /my/new/config.yml`
 Some more information on the config can be found [here](pkg/config/config.go).
 
+There are two built-in methods to facilitate the generation of blessclient configs:
+
 #### Init
 
 `blessclient init` will ask you some questions in order to assemble some basic configuration.
 
 #### Import-config
 Alternatively, you can also use pre-generated config files.
 
-`blessclient import-config http://github.com/..../teamA_blessclient.yml`
+`blessclient import-config git@github.com:/..../teamA/blessconfig.yml`
```

Comment:
can we maybe give a few examples here, so that it is clear that it is not just git+ssh?